c/validator_utils.h

```diff
@@ -549,7 +549,7 @@ int sys_recover_account(gw_context_t *ctx, uint8_t message[32],
     }
 
     /* found script, recover account script */
-    if (*script_len < script_seg.size) {
+    if (0 != *script_len && *script_len < script_seg.size) {
```

Comment:
Why? what happend when set script_len to 0?

Reply:
Simply because polyjuice pass 0 here and this check always return false.